Sources/VaporAPNS/APNSContainers.swift

```diff
@@ -50,7 +50,90 @@ public final class APNSContainers: Sendable {
 }
 
 extension APNSContainers {
-
+    /// Configure APNs for a given container ID.
+    ///
+    /// You must configure at lease one client in order to send notifications to devices. If you plan on supporting both development builds (ie. run from Xcode) and released builds (ie. TestFlight/App Store), you must configure at least two configurations:
+    ///
+    /// ```swift
+    /// let apnsKey = ... // The .p8 file as a string.
```

Comment:
Let's add a note here to suggest passing it in as an environment variable and not hardcoding it in the source code

Reply:
Good callout, I’ll add a section for security considerations.

Reply:
Updated!